docs/examples/hypothesis_property_based_test.py

```diff
@@ -17,8 +19,24 @@ def test_property(instance):
     assert all('@' in email for email in instance.users)
 
 
-@given(st.builds(Model, price=st.floats(100, 200)))
+@given(st.builds(Model, Price=st.floats(100, 200)))
 def test_with_discount(instance):
     # This test shows how you can override specific fields,
     # and let Hypothesis fill in any you don't care about.
     assert 100 <= instance.price <= 200
+
+
+@given(st.builds(Model, price=st.floats(100, 200)))
+def test_with_discount_field_name(instance):
+    # This test shows how you can override specific fields,
+    # and let Hypothesis fill in any you don't care about.
+    assert 100 <= instance.price <= 200
+
+
+test_with_discount()
```

Comment:
I wouldn't call these functions, they're supposed to be pytest tests and therefore not called directly.

For the `test_with_discount_field_name` I would just add a comment saying it will fail and way.